phpunit.xml.dist

```diff
@@ -6,7 +6,6 @@
     bootstrap="vendor/autoload.php"
     colors="true"
     columns="max"
-    verbose="true"
```

Comment:
Why?

Reply:
This is intended. I’m disabling “verbose” by default so I can enable it in all test matrix except for the “repeat 100” one to reduce spam output because it’s not possible to disable verbosity once it’s set in the XML.